configure.ac

```diff
@@ -1482,14 +1482,12 @@ AC_MSG_RESULT( )
 AC_MSG_RESULT(Summary:)
 AC_MSG_RESULT(> Version: AC_PACKAGE_VERSION)
 
-AC_MSG_RESULT([> Required libraries])
+AC_MSG_RESULT([> Optional libraries])
 
 AC_MSG_RESULT([-> OpenSSL: $OPENSSL_PATH])
```

Comment:
Isn't OpenSSL required?

Reply:
Nope, optional in pretty much the same way as PCRE.

Reply:
https://github.com/cfengine/libntech/pull/63

Reply:
https://github.com/mendersoftware/mender-binary-delta/pull/43/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R48